miso-web/src/main/webapp/scripts/instituteDefaults_ajax.js

```diff
@@ -404,18 +417,33 @@ var Subproject = Subproject || {
     return select.join('');
   },
 
+  createReferenceGenomeSelect: function(idValue, referenceGenomeId) {
+    var select = [];
+    select.push('<select id="'+ idValue +'">');
+    select.push('<option value="-1">--Please select</option>');
```

Comment:
For subproject I wonder if we need an option to use the parent project reference? Rather than forcing the choice of a new reference.